internal/rollup/rollup_bundle.bzl

```diff
@@ -104,34 +112,46 @@ def run_rollup(ctx, sources, config, output):
   """
   map_output = ctx.actions.declare_file(output.basename + ".map", sibling = output)
 
+  _run_rollup(ctx, sources, config, output, map_output)
+
+  return map_output
+
+def _run_rollup(ctx, sources, config, output, map_output=None):
```

Comment:
it is in the private version of that function since for code-splitting there are multiple map files outputted.

The public API used by angular has not changed:

```
def run_rollup(ctx, sources, config, output):
  """Creates an Action that can run rollup on set of sources.

  This is also used by ng_package and ng_rollup_bundle rules in @angular/bazel.

  Args:
    ctx: Bazel rule execution context
    sources: JS sources to rollup
    config: rollup config file
    output: output file

  Returns:
    the sourcemap output file
  """
  map_output = ctx.actions.declare_file(output.basename + ".map", sibling = output)

  _run_rollup(ctx, sources, config, output, map_output)

  return map_output
```

but in the private `_run_rollup` function the map_output is option and the arguments sent to rollup vary depending on if it is set or not:

```
def _run_rollup(ctx, sources, config, output, map_output=None):
  args = ctx.actions.args()
  args.add(["--config", config.path])
  if map_output:
    args.add(["--output.file", output.path])
    args.add(["--output.sourcemap", "--output.sourcemapFile", map_output.path])
  else:
    args.add(["--output.dir", output.path])
    args.add(["--output.sourcemap"])
```